test/Microsoft.IdentityModel.Protocols.OpenIdConnect.Tests/ConfigurationManagerTests.cs

```diff
@@ -327,7 +327,7 @@ public void GetConfiguration()
             }
 
             // get configuration from https address, should throw
-            configManager = new ConfigurationManager<OpenIdConnectConfiguration>("https://someaddress.com", new OpenIdConnectConfigurationRetriever());
+            configManager = new ConfigurationManager<OpenIdConnectConfiguration>("https://someadress.com", new OpenIdConnectConfigurationRetriever());
```

Comment:
```suggestion
            configManager = new ConfigurationManager<OpenIdConnectConfiguration>("http://127.0.0.1", new OpenIdConnectConfigurationRetriever());
```

Reply:
matches this [PR](https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/commit/ec39ddb318b045ea1b0a2bd993356c6726e5a3bd) and resolves Josh's comment too.

Reply:
I'll push something that applies this to both parts of the test that refer to someaddress.com